package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-scrollspy",
-  "version": "2.6.3",
+  "version": "2.7.0",
```

Comment:
This was incremented for my own purposes until it can be integrated back into this repo. Thinking about it more I believe this could just be a patch release (`2.6.4`). As I believe it's backwards compatible because `rootEl` is optional. I can do more testing to confirm.